readme.md

```diff
@@ -0,0 +1,60 @@
+How to get started using the application
+Application dependencies
+More detailed information about how your application uses its data connections
+A link to an application website
+Licensing information
+
+# Getting started with CYF_Developer-Planner
```

Comment:
Is it CYF_Developer-Planner or CYF_Development-Planner? (Name of repo is different)

```suggestion
# Getting started with CYF_Development-Planner
```